benchmarks/build.gradle

```diff
@@ -0,0 +1,15 @@
+apply plugin: 'org.jetbrains.kotlin.platform.jvm'
```

Comment:
Shouldn't this just be `org.jetbrains.kotlin.jvm`? (without the `platform` part)

Reply:
You're right, it should be for this module